modules/ROOT/pages/6.6_Economic_Operator.adoc

```diff
@@ -411,7 +411,7 @@ _Table 38. Service provider, expected elements_
 |*cac:PostalAddress/cac:Country/cbc:IdentificationCode* |Code |1 |The code that identifies the country of the service provider. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr070-021[_tbr070-021_].
 
-*Rule*: The country of the service provider must always be specified. Compulsory use of the Code List link:{url-tree}/dist/cl/ods/[Country from EU Vocabularies, which is based in ISO].
+*Rule*: The country of the service provider must always be specified. Compulsory use of the Code List link:{url-tree-tmp}/dist/cl/ods/[Country from EU Vocabularies, which is based in ISO].
```

Comment:
Country from EU Vocabularies, which is based in ISO
I think this text should not be part of the link, only the word country